src/main/java/org/apache/sysds/runtime/controlprogram/paramserv/FederatedPSControlThread.java

```diff
@@ -337,6 +338,7 @@ protected void weighAndPushGradients(ListObject gradients) {
 
 		// Push the gradients to ps
 		_ps.push(_workerID, gradients);
+		//_ps.push(_workerID, modell)
```

Comment:
what's this? please conditionally on the configuration either push the gradients or model

Reply:
Thanks, the issue was fixed.